.github/workflows/conformance-ginkgo.yaml

```diff
@@ -0,0 +1,926 @@
+               export KERNEL=54
+            fi
+            export K8S_VERSION=${{ matrix.k8s-version }}
+            export CNI_INTEGRATION=kind
+            export INTEGRATION_TESTS=true
+            # GitHub actions do not support IPv6 connectivity to outside
+            # world.
+            export CILIUM_NO_IPV6_OUTSIDE=true
+            echo "/root/go/bin/ginkgo \
+             --focus=\"${{ steps.vars.outputs.focus }}\" \
+             --skip=\"${{ matrix.skip }}\" \
+             --seed=1679952881 \
+             -v -- \
+             -cilium.provision=false \
+             -cilium.image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+             -cilium.tag=${{ steps.vars.outputs.sha }}  \
+             -cilium.operator-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
+             -cilium.operator-tag=${{ steps.vars.outputs.sha }} \
+             -cilium.hubble-relay-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci \
+             -cilium.hubble-relay-tag=${{ steps.vars.outputs.sha }} \
+             -cilium.kubeconfig=/root/.kube/config \
+             -cilium.provision-k8s=false \
+             -cilium.operator-suffix=-ci"
+
+              ./test.test \
+               --ginkgo.focus="${{ steps.vars.outputs.focus }}" \
+               --ginkgo.skip="${{ matrix.skip }}" \
+               --ginkgo.seed=1679952881 \
+               --ginkgo.v -- \
+               -cilium.provision=false \
+               -cilium.image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+               -cilium.tag=${{ steps.vars.outputs.sha }}  \
+               -cilium.operator-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
+               -cilium.operator-tag=${{ steps.vars.outputs.sha }} \
+               -cilium.hubble-relay-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci \
+               -cilium.hubble-relay-tag=${{ steps.vars.outputs.sha }} \
+               -cilium.kubeconfig=/root/.kube/config \
+               -cilium.provision-k8s=false \
+               -cilium.operator-suffix=-ci
+
+      - name: Fetch artifacts
+        if: ${{ !success() }}
+        uses: cilium/little-vm-helper@0b7d7157dae56a44dc531c852e7756dc671071af # v0.0.3
+        with:
+          provision: 'false'
+          cmd: |
+            cd /host
+            kubectl get pods --all-namespaces -o wide
+            ./cilium-cli status
+            mkdir -p cilium-sysdumps
+            tar -zcf test_results-${{ join(matrix.*, '-') }}.tar.gz /host/test/test_results
+            ./cilium-cli sysdump --output-filename cilium-sysdump-${{ join(matrix.*, '-') }}-final || true
+
+      - name: Upload artifacts
+        if: ${{ !success() }}
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: cilium-sysdumps
+          path: |
+            cilium-sysdump-*.zip
+            bugtool-*.tar.gz
+            test_results-*.tar.gz
+          retention-days: 5
```

Comment:
Where are these uploaded? I can't see any in the case of the failing test: https://github.com/cilium/cilium/actions/runs/5090956330/jobs/9150414890. Did I miss some page?

Reply:
They are uploaded to the summary page of the job run here: https://github.com/cilium/cilium/actions/runs/5090956330

Special note that previous attempts, which the link that you have posted is pointing to attempt [#1](https://github.com/cilium/cilium/actions/runs/5090956330/attempts/1), will have their attachments removed once a new attempt is triggered. Also if we re-trigger a job failure, which is what I did with the Kafka test, the attachments of the previous attempts for successful jobs will be "transferred" to the new attempt.

Reply:
Uh, wait. You don't collect full ginkgo artifacts anymore, just sysdumps? That's not quite the same. In sysdumps, we don't have journalctl, dmesg, or test-output.log.

I can't believe I missed that for the Runtime tests :grimacing:

Reply:
Which ginkgo artifacts are you referring to? You mean the `test_results-*.tar.gz`? That's part of the `cilium-sysdumps.zip` artifact.

Reply:
Nevermind. I don't have a lot of time to check and compare the exact list, so let's forget about it and I'll send fixes when/if I need to debug a test.